server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -158,7 +199,7 @@ private boolean isCurrentSnapshotsOnly(String[] snapshots) {
         return (snapshots.length == 1 && GetSnapshotsRequest.CURRENT_SNAPSHOT.equalsIgnoreCase(snapshots[0]));
     }
 
-    private List<SnapshotInfo> buildSimpleSnapshotInfos(final Set<SnapshotId> toResolve,
+    private List<SnapshotInfo> buildSimpleSnapshotInfos(final String repository, final Set<SnapshotId> toResolve,
```

Comment:
Parameter `repository` isn't used here + this method could be made static.

Reply:
https://github.com/elastic/elasticsearch/pull/42090/commits/ca011e388ecab53e3a11c16009c9b8212919f695